src/Widgets/SaveDialog.vala

```diff
@@ -24,6 +24,9 @@ public class Screenshot.SaveDialog : Granite.Dialog {
     public Gdk.Pixbuf pixbuf { get; construct; }
     public Settings settings { get; construct; }
 
+    private Granite.ValidatedEntry name_entry;
+    private Gtk.Button save_btn;
```

Comment:
You do not need these members.  You can just use local variables in `construct {}`.